base_layer/wallet_ffi/src/lib.rs

```diff
@@ -5473,6 +5486,7 @@ pub unsafe extern "C" fn wallet_create(
     let peer_seeds = PeerSeedsConfig {
         dns_seeds_name_server: DEFAULT_DNS_NAME_SERVER.parse().unwrap(),
         dns_seeds_use_dnssec: true,
```

Comment:
From my chat based experience- dnssec needs to be turned off or all our DNS validations will fail. 

This is the chat config for PeerSeeds which *does* use DNS (including the existing integrated wallet branch)

```
        peer_seeds: PeerSeedsConfig {
            dns_seeds_use_dnssec: false,
            dns_seeds_name_server: DEFAULT_DNS_NAME_SERVER.parse().unwrap(),
            dns_seeds: StringList::from(vec![format!("seeds.{}.tari.com", network.as_key_str())]),
            ..PeerSeedsConfig::default()
        },
        ```

Reply:
yip mapped that through